source/client-side-encryption/client-side-encryption.md

```diff
@@ -11,11 +11,42 @@ ______________________________________________________________________
 MongoDB 4.2 introduced support for client side encryption, guaranteeing that sensitive data can only be encrypted and
 decrypted with access to both MongoDB and a separate key management provider (supporting AWS, Azure, GCP, a local
 provider, and KMIP). Once enabled, data can be seamlessly encrypted and decrypted with minimal application code changes.
-6.0 introduced the next generation of client side encryption based on a Structured Encryption framework which allows
+7.0 introduced the next generation of client side encryption based on a Structured Encryption framework which allows
 expressive encrypted search operations. This spec covers both capabilities - 1st generation, "Client Side Field Level
 Encryption" and generation 2, "Queryable Encryption" - as the associated core cryptographic library and supporting
 drivers share a common codebase.
 
+### Naming
+
+The public name of this feature is
+[In-Use Encryption](https://www.mongodb.com/docs/manual/core/security-in-use-encryption/) and consists of:
+
+- Client-Side Field Level Encryption (CSFLE).
+- Queryable Encryption (QE).
+
+Internally, CSFLE is sometimes called Client Side Encryption (like this specification). In-Use Encryption is sometimes
+called Field Level Encryption (FLE). CSFLE and QE are sometimes called FLE1 and FLE2, respectively.
```

Comment:
Very minor, but this could repeat "internally," like it repeats "sometimes called" (or periods to commas) - if that is the intent. And in-use (being mentioned first earlier) could come before CSFLE.

Reply:
I like that idea. Reworded to move In-Use Encryption first, and repeat use of "Internally".